g3w-admin/client/views.py

```diff
@@ -216,7 +216,10 @@ def render_to_response(self, context, **response_kwargs):
 
         # Add G3W_CLIENT_COOKIE_SESSION_TOKEN cookie to response
         response = super().render_to_response(context)
-        response.set_cookie(settings.G3W_CLIENT_COOKIE_SESSION_TOKEN, secrets.token_hex(16), samesite='None')
+
+        # Only with https set samesite='None' for cross-site requests, i.e. for cross-site iframe
+        kwargs = {'samesite': 'None'} if self.request.is_secure() else {'samesite': 'Strict'}
```

Comment:
Locally it no longer shows the error, but in case of a **https** installation, I think you are still missing the `secure` field, ref: https://github.com/g3w-suite/g3w-admin/issues/791#issuecomment-2019526871